### PR TITLE
07 post api articles article id comments

### PR DESCRIPTION
**Pull request by Meli-Nzekele**
app.test.js: 07 POST Tests - Lines 151 to 196
app.test.js: Error handling Tests - Lines 252 to 313

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -203,5 +249,67 @@ describe("app", () => {
...
+        it("404: responds with correct message for valid but non-existent article_id", () => {
+          const testNewComment = {
+            username: "butter_bridge",
+            body: "New Comment",
+          };
+
+          return request(app)
+            .post("/api/articles/4321/comments")
+            .send(testNewComment)
+            .expect(400)
```
Comment (Keb84) [inline __tests__/app.test.js]: This should be a 404 - non existent ID

File: __tests__/app.test.js [hunk 2]
```diff
@@ -203,5 +249,67 @@ describe("app", () => {
...
+
+          return request(app)
+            .post("/api/articles/4/comments")
+            .send(testNewComment)
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toBe("Invalid Comment Format");
+            });
+        });
+      });
```
Comment (Keb84) [inline __tests__/app.test.js:335]: Can you add another test here for 201 - ignores invalid properties? 🤔

**Review by Keb84 — APPROVED** [review]
Have a look at adding 1 more test as mentioned in the comments, otherwise good to go 👍

File: __tests__/app.test.js (hunk 2 above)
Comment (Meli-Nzekele) [reply __tests__/app.test.js:335]: No problem :)

File: __tests__/app.test.js (hunk 1 above)
Comment (Meli-Nzekele) [reply __tests__/app.test.js]: Thank you, I will update this :)

**Pull request merged**